vite-plugin-transform-imports.js

```diff
@@ -0,0 +1,14 @@
+export default function transformImports() {
```

Comment:
Is it possible we can setup the resolution delegation to window.Vue instance in ComfyUI_frontend, instead of requiring each extension to perform this build-time hack?

Requiring each extension to do that can lead to issues if we want to expose more libraries such as threejs or primevue later, we need to go to each extension's setup an add extra logic there, which can be a lot of work both for us and extension developers.

Reply:
I need this because a issue of Vue while building in custom node, I will show later in a call